libyara/modules/math/math.c

```diff
@@ -27,6 +27,8 @@ ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
+#include <stdlib.h>
+#include <sys/errno.h>
```

Comment:
`#include <errno.h>` should be enough, `#include <sys/errno.h>` is not being found and CI builds are failing.